### ISSUE TITLE
GeoTIFF get raw values from DSM by click

### ISSUE THREAD

**Issue by undefinedSolutions** [openlayers/openlayers#13389]
**Is your feature request related to a problem? Please describe.**
I do have an DSM-COG at the URL (https://data.masterarbeit-cog.de/cog/dsm_strenzfeld.tif) and I was able to put in on a map with an color-Range with this code:
```
const sourceCOG = new GeoTIFF({
  normalize: false,
  sources: [
    {
      url: 'https://data.masterarbeit-cog.de/cog/dsm_strenzfeld.tif',
    },
  ],
});
sourceCOG.setAttributions(AoiStrenzfeld.attributions);
console.log(sourceCOG.getRevision())
const layer = new TileLayer({
  source: sourceCOG,
  style: {
    color: [
      'interpolate',
      ['linear'],
      ['band', 1],
      // Colors
      57,
      [0, 0, 0, 0],
      57,
      [44, 129, 184],
      70.75,
      [171, 223, 167],
      84.5,
      [255, 255, 195],
      98.25,
      [253, 178, 103],
      112,
      [215, 16 ,29],
    ],
  },
});
```

And I am able to get RGBA Values from Pixels with this:
```
map.on("click", (evt) => {
  map.forEachLayerAtPixel(
    evt.pixel,
    function (layer, pixel) {
      if(layer.getSource() instanceof GeoTIFF) {
        console.log(pixel[0],pixel[1],pixel[2],pixel[3]);
      }
    }
  );
});
```

But I was not able to find a solution to get the war highValues from the GeoTIFF itself.
I Found this ticket, but there the hight is calculated by the RGBA Values: https://github.com/openlayers/openlayers/issues/12833#issuecomment-934585869 but this GeoTIFF has RGB Values what Worked...

**Describe the solution you'd like**
I am searching for a solution to get the raw Values by click on map from Band 1 of this [GeoTIFF](https://data.masterarbeit-cog.de/cog/dsm_strenzfeld.tif) what has as Data type Float32 if this is important...

**Comment by mike-000** [comment]
Already implemented in the latest dev version via #13338

**Comment by undefinedSolutions** [comment]
Oh! Thank you, I was not able to find this -.- sorry
